tests/sync/genesis/test_genesis_voting.py

```diff
@@ -0,0 +1,84 @@
+import time
+import pytest
+from pytest_testconfig import config as testconfig
+
+import tests.queries as q
+from tests.sync.test_sync import new_client_in_namespace
+from tests.test_bs import get_conf, setup_bootstrap, start_poet, add_curl
+from tests.utils import print_hits_entry_count
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+
+# set a X nodes cluster
+# sleep in order to enable a block to be delivered
+# validate a block has been delivered
+# add an additional client while still in genesis
+# sleep until layer 4
+# validate no errors occurred
+# validate new node didn't receive any new blocks before being synced
+# @pytest.mark.xfail(strict=True, reason="new pod creates blocks before being synced")
+def test_unsync_while_genesis(init_session, setup_bootstrap, start_poet, add_curl):
+    time_before_first_block = 70
```

Comment:
shouldn't this be a function of layer duration + genesis waiting time?

Reply:
it should, right now it's hard coded and synced with the suite arguments.
I'll consult with Barak about this.

Reply:
fixed in 5c6b63aabad9a2849d5c4dcc7755c9ab121a0579